java/src/apps/gui/GuiLafPreferencesManager.java

```diff
@@ -293,7 +293,7 @@ public void applyLookAndFeel() {
      * for each.
      */
     private void applyFontSize() {
-        logAllFonts();
+        if (log.isDebugEnabled()) logAllFonts();
```

Comment:
I'm sorry that you still don't understand my use case for this.

"going to be reading the code to understand why it's not logging at the debug level" doesn't bother me.  I would have had to do that anyway without this change.  I care about the other case: Now I don't have to read that routine if I know I'm not logging DEBUG.

If the "going to be reading the code to understand why it's not logging at the debug level" case bothers you, you could add a logging level as an argument.